### PR TITLE
Document how to auto-run flows by monitoring filesystem events

### PR DESCRIPTION
**Pull request by kurtmckee**
Documentation
-------------

-   [[sc-16711]](https://app.shortcut.com/globus/story/16711)     Document how to automatically run a flow based on filesystem change events.

### CODE REVIEW

**Review by jakeglobus — APPROVED** [review]
This is cool, nice job!

File: docs/source/automating-flow-runs.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Shell scripting
+===============
+
+`watchdog`_ provides a CLI tool named ``watchmedo``.
+If you're comfortable with shell scripting
+then you can use the following command to run a script named ``runner.sh``.
+
+
+..  literalinclude:: code_snippets/watchmedo.sh
+    :caption: ``watchmedo.sh`` [:download:`download <code_snippets/watchmedo.sh>`]
```
Comment (sirosen) [inline docs/source/automating-flow-runs.rst:19]: Is the `:download:` role coming from a sphinx extension? I haven't seen this before but it seems _really nice_.

Comment (kurtmckee) [reply docs/source/automating-flow-runs.rst:19]: I _think_ it's a built-in role...but I'm not sure!

Comment (sirosen) [reply docs/source/automating-flow-runs.rst:19]: [Looks](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html) like it is! I must have missed it when it was added... before I started using sphinx... 😅 
image

**Pull request merged**